resources/public/api/conf/1.0/application.raml

```diff
@@ -200,9 +204,11 @@ traits:
       get:
         displayName: Get a Movement Arrival for an arrival ID
         description: |
-         Get a specific Movement Arrival. Any Movement Arrivals more than 31 days old will have been archived along with their related messages. Archived movements will return an HTTP 404 Not Found status.
+          Retrieve a specific transit movement arrival that is 31 days old or younger.
 
-          You will only be able to retrieve these messages if you have the correct Economic Operator Registration and Identification (EORI) number for that Movement Arrival.
+          Movements older than 31 days and their related messages are archived and cannot be returned by the API.
+
+          **Note:** You must use the correct Economic Operators Registration and Identification (EORI) number for a movement to retrieve any messages relating to that movement.
```

Comment:
I realise you've been more rewording this, as opposed to completely overhauling it, but this is worth a mention.

You can only see a movement if your **enrolment** EORI matches the enrolment EORI that created the notification -- which isn't supplied in this call by the trader, but extracted from the auth data based on the bearer token -- i.e. it's to do with who you're authenticated as.

I'm not sure how to word this, and I don't want to hold this up trying to work out what to say necessarily as what you've written matches what is there now, but it might be something we should ticket and think about separately.

Reply:
I agrees that this is important and should be captured in the public API documentation.

We might have to document this as follows:

- Add an Authentication section at the start of https://developer.service.hmrc.gov.uk/api-documentation/docs/api/service/common-transit-convention-traders/1.0 that outlines how API authentication works (insert section after overview or versioning sections).
- Mention EORI enrolment in request header text (Authorisation) for all endpoints.
- Reword EORI text in each endpoint description.